frontend/package.json

```diff
@@ -110,7 +110,6 @@
       "eslint-plugin-react": "7.23.0",
       "eslint-plugin-react-hooks": "4.2.0",
       "graphql-tag": "2.12.6",
-      "graphqurl": "1.0.1",
```

Comment:
It looks like we use the cli `gq` version here:
https://github.com/iFixit/react-commerce/blob/d528e7820d58d4f6f1ea3fd6e1f14e40ebf42f2b/frontend/lib/shopify-storefront-sdk/generate-schema.js#L20

Found it by running `git grep "gq "`

Reply:
OMG, thank you... this is the the second time I've done this exact thing. Derp. Will fix.